.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test cibuildwheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
actions/checkout@v2 seems to be the current version, maybe go with that?

Reply:
Note that version 2 doesn’t support submodules. Just an aside.

Reply:
OK, forget my comment then, v1 works :)